src/lib/ppx_coda/versioned_module.ml

```diff
@@ -383,17 +371,180 @@ let version_module ~loc ~path:_ modname modbody =
     Format.(fprintf err_formatter "%s@." (Printexc.get_backtrace ())) ;
     raise exn
 
+(* code for module declarations in signatures
+
+   - add deriving bin_io, version to list of deriving items for the type "t" in versioned modules
+   - add "module Latest = Vn" to Stable module
+ *)
+
+let convert_module_type_signature_item sigitem =
+  match sigitem.psig_desc with
+  | Psig_type
+      (recflag, [({ptype_name= {txt= "t"; loc}; ptype_attributes; _} as type_)])
+    ->
+      let module E = Ppxlib.Ast_builder.Make (struct
+        let loc = loc
+      end) in
+      let open E in
+      let derivings, other_attrs =
```

Comment:
Can you combine this with `add_deriving` above? It looks like here and there are doing the same thing slightly differently, would be good to de-duplicate.

Reply:
Done.